spec/bowling_scorecard_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'bowling_scorecard.rb'
+
+describe BowlingScorecard do
+  describe '.add_knocked_pins' do
+    it 'registers number of pins knocked in a roll' do
```

Comment:
If you were refactoring, I wonder if you could get rid of this test because your third test tests this method.